### PR TITLE
Framelooping ImageView invalidates Framebuffer

### PR DESCRIPTION
**Pull request by purvisa-at-google-com**

### CODE REVIEW

File: gapis/api/vulkan/frame_loop.go [hunk 1]
```diff
@@ -1210,6 +1210,12 @@ func (f *frameLoop) resetImageViews(ctx context.Context, stateBuilder *stateBuil
 		// Write the commands needed to recreate the destroyed object
 		imageView := GetState(f.loopStartState).imageViews.Get(toCreate)
 		stateBuilder.createImageView(imageView)
+
+		franebufferUsers := imageView.FramebufferUsers().All()
+		for framebuffer, _ := range franebufferUsers {
```
Comment (hevrard) [inline gapis/api/vulkan/frame_loop.go]: typo franebufferUsers -> fra**m**ebufferUsers

File: gapis/api/vulkan/frame_loop.go [hunk 2]
```diff
@@ -144,8 +144,8 @@ type frameLoop struct {
 	eventChanged   map[VkEvent]bool
 	eventDestroyed map[VkEvent]bool
 
-	framebuffersToDestroy map[VkFramebuffer]bool
-	framebuffersToCreate  map[VkFramebuffer]bool
+	framebufferToDestroy map[VkFramebuffer]bool
```
Comment (pau-baiget) [inline gapis/api/vulkan/frame_loop.go]: I'm wondering why moving from plural to singular on this variable? Still a list of framebuffers, right?

Comment (purvisa-at-google-com) [reply gapis/api/vulkan/frame_loop.go]: Just for uniformity with the other maps. It was easier to move the outlier into the group than move the group to the outlier.

File: gapis/api/vulkan/frame_loop.go (hunk 1 above)
Comment (purvisa-at-google-com) [reply gapis/api/vulkan/frame_loop.go]: fixed

**Pull request merged**